src/internet_identity/tests/archive_integration_tests_pull.rs

```diff
@@ -262,7 +262,7 @@ mod pull_entries_tests {
             operation: Operation::RegisterAnchor {
                 device: DeviceDataWithoutAlias {
                     pubkey: device_data_1().pubkey,
-                    credential_id: None,
+                    credential_id: device_data_1().credential_id,
```

Comment:
out of curiosity, why did this change?

Reply:
I introduced a credential id on the `device_data_1()` test data (to make it a proper webauthn device), so the archive test that uses it now also archives the credential id.